src/app/models/rex-release.ts

```diff
@@ -0,0 +1,52 @@
+import memoize from 'lodash/memoize';
```

Comment:
prettier made this too different to be just a change, but there are no real changes.
You can see the [commit](https://github.com/openstax/os-webview/pull/2701/commits/1c72e0e448a60ec6c66868ca5bb9e5342f8c5021#diff-d60eda19c473d0611d4a5fe9ed4f07a4867a1aebdb60d1bf444d15a580faf8bb) before the prettier commit.

Reply:
I wonder, does committing just the extension rename first, then making the content changes preserve the link? It's so hard to review these because github doesn't let us adjust the similarity threshold

Reply:
If you look at the individual commit, you would see what you expect to: the rename (only), then the changes to the new file in the following commit. If you look at the "Files changed" you will see the total changes, governed by the similarity threshold. That's why I kept all the intermediate commits and linked to them where I thought it would help the review process.

Reply:
Looks like the commit link got horked? See if this works. Otherwise, you can generally look at the Commits and see from their names which file(s) they include.
https://github.com/openstax/os-webview/pull/2701/commits/c293d19dcbd23273f7dc5e4a3a29e00a29269ecd#diff-d60eda19c473d0611d4a5fe9ed4f07a4867a1aebdb60d1bf444d15a580faf8bb